Dockerfile

```diff
@@ -2,16 +2,21 @@ FROM java:8-jre
 
 MAINTAINER PalSzak
 
-RUN  wget -q -O - http://www.eu.apache.org/dist/kafka/0.8.2.1/kafka_2.10-0.8.2.1.tgz | tar -xzf - -C /opt
+ENV SCALA_VERSION 2.11
+ENV KAFKA_VERSION 0.9.0.1
 
-ENV PATH /opt/kafka_2.10-0.8.2.1/bin:$PATH
+RUN  wget -q -O - http://www.eu.apache.org/dist/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz | tar -xzf - -C /opt
+
+RUN mv /opt/kafka_$SCALA_VERSION-$KAFKA_VERSION /opt/kafka
```

Comment:
I [merged](https://github.com/DockerKafka/kafka-docker/commit/8f0c0a4d2e12e69685f44c4ef76bda04efbca1b5) this two RUN command. It makes the image 30MB smaller.